### ISSUE TITLE
datetime UTC has 7 digits, not 6. Not easily compatible with python3 after sqlite3 queries

### ISSUE THREAD

**Issue by MrCee** [Radarr/Radarr#8658]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Running queries in sqlite3 for the date a movie was added returns the UTC result having microseconds consist of 7 digits
I'm pretty sure the 7 digit microseconds should be reduced to 6 digit microseconds

2023-04-04 14:57:44.2693676Z (Radarr 7 digits before 'Z')
2023-04-04 14:57:44.2693676Z (Radarr 7 digits before 'Z')
2023-06-05 05:14:51.4386289Z (Radarr 7 digits before 'Z')
2023-05-27 07:27:07.336155Z (Sonarr 6 digits before 'Z')
2023-03-24 01:07:10.175149Z (Sonarr 6 digits before 'Z')

datetime.datetime.utcnow() also only has 6 digits....
datetime.datetime(2023, 6, 5, 5, 39, 8, 400526)

### Expected Behavior

6 digits for microseconds should be the maximum

### Steps To Reproduce

movies = subprocess.run(['sqlite3', str(radarr_db), 'select path, added from Movies;'],
                        capture_output=True, text=True).stdout.strip().split('\n')

Movies shows path & added and the added date has 7 microsecond digits

### Environment

```markdown
- OS: MacOS latest
- Docker Install: linuxserver/ radarr:latest
```

### What branch are you running?

Master

### Trace Logs?

.

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [X] I have followed the steps in the wiki link above and provided the required trace logs that are relevant and show this issue.

**Comment by Qstick** [comment]
I assume you aren't using Sonarr v4 yet? It's likely same as Radarr

**Comment by MrCee** [comment]
> I assume you aren't using Sonarr v4 yet? It's likely same as Radarr

That is correct. sonarr UTC datetime works perfectly with python3 and radarr datetime does not due to extra digits in UTC being 7 digits in length before the 'Z' which is not compliant with python3 as per the docs.

I'm merging results from 2 seperate queries here and it's apparent that radarr should not have 7 digit's and extra effort is involved to truncate back to 6 digits to allow python3 to work perfectly.

If we can get this small but important factor aligned... I'd probably have saved 4+ hours trying to understand why and extra digit was causing my queries to fail. 

Can we please remove the 7th digit from UTC?

**Comment by Qstick** [comment]
The good news is we aren't trying to be compliant with python3, so that spec means nothing.

Well look at the difference here.

**Comment by Qstick** [comment]
Essentially we store values at 100 nanosecond precision (a tick in dotnet speak). This is likely due to the mapper change in v4 (which will also be coming soon to Sonarr)

Below is a good summary of this issue

https://nickb.dev/blog/iso8601-and-nanosecond-precision-across-languages/

If you want a library that can read these higher precision iso 8601 compliant strings in python, use numpy. But given these are fully compliant for SQLite and ISO, I've got no plans to look at this
